homework/linux.txt

```diff
@@ -0,0 +1,35 @@
+124:gdm          883  0.0  0.0 1114420 4308 ?        S<sl 08:05   0:00 /usr/bin/pulseaudio --daemonize=no --log-target=journal
+129:whoopsie     904  0.0  0.0 326848  4132 ?        Ssl  08:05   0:00 /usr/bin/whoopsie -f
+132:gdm          919  0.0  0.0   7400  1504 ?        Ss   08:05   0:00 /usr/bin/dbus-daemon --session --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
+150:gdm         1107  0.0  0.0   7116   844 tty1     S+   08:05   0:00 /usr/bin/dbus-daemon --config-file=/usr/share/defaults/at-spi2/accessibility.conf --nofork --print-address 3
+151:gdm         1128  0.0  0.8 3855260 72480 tty1    Sl+  08:05   0:13 /usr/bin/gnome-shell
+158:gdm         1172  0.0  0.0 2671020 7320 tty1     Sl+  08:06   0:00 /usr/bin/gjs /usr/share/gnome-shell/org.gnome.Shell.Notifications
+180:raxonpc     1435  5.2  0.1 2953584 12880 ?       S<sl 08:06  34:19 /usr/bin/pulseaudio --daemonize=no --log-target=journal
+182:raxonpc     1440  0.0  0.0  10764  5248 ?        Ss   08:06   0:06 /usr/bin/dbus-daemon --session --address=systemd: --nofork --nopidfile --systemd-activation --syslog-only
+186:raxonpc     1473  0.0  0.0 251472  4104 ?        SLl  08:06   0:01 /usr/bin/gnome-keyring-daemon --daemonize --login
+193:raxonpc     1512  0.0  0.0 175180  2208 tty2     Ssl+ 08:06   0:00 /usr/lib/gdm3/gdm-x-session --run-script env GNOME_SHELL_SESSION_MODE=ubuntu /usr/bin/gnome-session --systemd --session=ubuntu
+196:raxonpc     1568  0.0  0.0 251056  1720 ?        Sl   08:06   0:00 /usr/bin/gnome-keyring-daemon --start --foreground --components=secrets
+197:raxonpc     1617  0.0  0.0   6040    36 ?        Ss   08:06   0:00 /usr/bin/ssh-agent /usr/bin/im-launch env GNOME_SHELL_SESSION_MODE=ubuntu /usr/bin/gnome-session --systemd --session=ubuntu
+199:raxonpc     1639  0.0  0.0   7380  1312 ?        S    08:06   0:01 /usr/bin/dbus-daemon --config-file=/usr/share/defaults/at-spi2/accessibility.conf --nofork --print-address 3
+202:raxonpc     1665  6.8  4.4 4838692 359972 ?      Ssl  08:06  44:48 /usr/bin/gnome-shell
+212:raxonpc     1741  0.0  0.1 2671332 12936 ?       Sl   08:06   0:01 /usr/bin/gjs /usr/share/gnome-shell/org.gnome.Shell.Notifications
+236:raxonpc     1913  0.0  1.9 1353356 155348 ?      Sl   08:06   0:09 /snap/snap-store/518/usr/bin/snap-store --gapplication-service
+292:raxonpc    32912  0.0  0.7 1265012 58708 ?       Sl   17:58   0:00 /usr/bin/gnome-calendar --gapplication-service
+293:raxonpc    32916  0.0  0.5 479396 44600 ?        SLl  17:58   0:00 /usr/bin/seahorse --gapplication-service
+  376  mkdir dupa
+  377  touch processes.txt
+  378  ps aux > processes.txt
+  379  cp processes.txt copy.txt
+  380  mkdir backup
+  381  mv copy.txt backup
+  382  wc -l processes.txt
+  383  wc -l processes.txt > selected.txt
+  384  grep -ne "/bin" processes.txt >> selected.txt

```

Comment:
```suggestion
  384  grep -ne /bin processes.txt >> selected.txt
```
Your solution is correct, but my script do not cover cases with quote marks. I have updated it, so either you need to rebase your fork of just accept this suggestion to make it pass 😉